docs/design/2021-04-21-unify-log-library.md

```diff
@@ -0,0 +1,141 @@
+# Proposal: Unifying Log Library
+
+- Author(s): [Yifan Xu](https://github.com/SabaPing)
+- Last updated:  Apr 12, 2021
+- Discussion at: https://github.com/pingcap/tidb/pull/24181
+- Tracking issue: https://github.com/pingcap/tidb/issues/24190
+
+## Motivation or Background
+
+Except for slow query logs, all other logs must satisfy the [unified-log-format RFC standard](https://github.com/tikv/rfcs/blob/master/text/2018-12-19-unified-log-format.md).
+
+However, in practice, it was found that the format of logs is confusing:
+
+- There are few logging configuration instructions in the document. We need to enrich document especially the type and the format of logs each component would emit.
```

Comment:
How would "Unify Log Library" change this situation?

Reply:
I have added a new doc task to address this problem. See https://github.com/pingcap/tidb/issues/24190#issuecomment-837991412